sklearn/linear_model/coordinate_descent.py

```diff
@@ -470,7 +473,10 @@ def enet_path(X, y, l1_ratio=0.5, eps=1e-3, n_alphas=100, alphas=None,
         if dual_gap_ > eps_:
             warnings.warn('Objective did not converge.' +
                           ' You might want' +
-                          ' to increase the number of iterations',
+                          ' to increase the number of iterations.' +
+                          ' Fitting data with alpha near zero,' +
```

Comment:
Did we not agree to add this message only when `alpha` is less than some heuristic value?